File: 2st/Riyeon/OX퀴즈.py
def solution(quiz):
    answer = []
    for i in quiz:
        li = i.split(' ')
        num = 0
        if li[1] == '+':
            num = int(li[0]) + int(li[2])
        else:
            num = int(li[0]) - int(li[2])
        if num == int(li[4]):
            answer.append('O')
        else:
            answer.append('X')
    return answer

File: 3st/Juhee/OX퀴즈.java
class Solution {
    public String[] solution(String[] quiz) {
		String[] answer = new String[quiz.length];
		for (int i = 0; i < quiz.length; i++) {

			String arr[] = quiz[i].split(" ");

			int left = arr[1].equals("+") ? Integer.parseInt(arr[0]) + Integer.parseInt(arr[2])
					: Integer.parseInt(arr[0]) - Integer.parseInt(arr[2]);
			int right = Integer.parseInt(arr[4]);

			answer[i] = left == right ? "O" : "X";
		}
		return answer;
	}
}
